scripts/report-reproducibility-pilot.py

```
@@ -0,0 +1,249 @@
+    }
+
+    report_path = output_dir / "report.json"
+    summary_path = output_dir / "summary.md"
+    report_path.write_text(json.dumps(report, indent=2) + "\n", encoding="utf-8")
+
+    summary_lines = [
+        "# Reproducibility Pilot Summary",
+        "",
+        f"- Image: `{image_name}`",
+        f"- Status: `{status}`",
+        f"- Comparison basis: `oci_manifest_digest`",
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)